packages/frontend/core/src/components/affine/setting-modal/general-setting/appearance/index.tsx

```diff
@@ -98,51 +104,10 @@ export const AppearanceSettings = () => {
             />
           </SettingRow>
         ) : null}
-        {BUILD_CONFIG.enableNewSettingUnstableApi && BUILD_CONFIG.isElectron ? (
-          <SettingRow
-            name={t['com.affine.appearanceSettings.windowFrame.title']()}
-            desc={t['com.affine.appearanceSettings.windowFrame.description']()}
-          >
-            <RadioGroup
-              items={windowFrameStyleOptions.map(option => ({
-                value: option,
-                label:
-                  t[`com.affine.appearanceSettings.windowFrame.${option}`](),
-              }))}
-              value={appSettings.windowFrameStyle}
-              className={settingWrapper}
-              width={250}
-              onChange={(value: AppSetting['windowFrameStyle']) => {
-                updateSettings('windowFrameStyle', value);
-              }}
-            />
-          </SettingRow>
-        ) : null}
```

Comment:
why removed this and line 125?

Reply:
I think these two options currently have no effect?

Reply:
WOW what are these